.github/workflows/main.yml

```diff
@@ -1,19 +1,21 @@
 name: noop run
 
 on:
-  - pull_request
-  - push
+  pull_request:
+  push:
+    branches:
+      - master
 
 jobs:
   unit:
     runs-on: ubuntu-latest
     name: Run msync --noop against all modules
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - name: Setup ruby
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: 3.0
+          ruby-version: 3
```

Comment:
will that use the latest 3.x release? Don't you want a specific minor release like `"3.4"`?

Reply:
I wasn't sure. I kind of like just using the latest since then we don't need to update it. I'd rather spend time fixing compatibility with the latest than to keep some value up to date. Our use of modulesync is also really light.

Reply:
it already does, as `3.0` is not a string, but a float, which is… `3`, so that change is just making it more explicit

Reply:
Exactly what I wrote in my commit message :)